Jenkinsfile

```diff
@@ -78,12 +78,7 @@ pipeline {
        stage ('Alias Test') {
             // only run this stage when triggered by a cron timer and the commit does not have []skip-ci in the message
             // for example, only run integration tests during the timer triggered nightly build
-            when {
-                allOf {
-                    triggeredBy 'TimerTrigger'
-                    branch "master"
-                }
-            }
+            when { branch 'WDT-alias-test-branch' }
```

Comment:
the comments that precede this when clause do not match the contents of the when-test.  Which one is correct?

Reply:
You caught this while I am still making the changes we discussed in meeting. This is temporary. Although for some reason its not working. My build skipped the alias test step on check in